docs/tooling/cli-guides/setup-a-devnet.md

```diff
@@ -758,3 +758,144 @@ curl http://52.204.202.216:9650/ext/bc/JjDfmxM3hAEX3VuaKH4PpQskhrvp2pzGTgYLpDwin
+            "encrypt-connection": false,
+            "account-private-key": "<ACCOUNT_PRIVATE_KEY>"
+        },
+        {
+            "subnet-id": "giY8tswWgZmcAWzPkoNrmjjrykited7GJ9799SsFzTiq5a1ML",
+            "blockchain-id": "JjDfmxM3hAEX3VuaKH4PpQskhrvp2pzGTgYLpDwinMzFeHJYA",
+            "vm": "evm",
+            "api-node-host": "127.0.0.1",
+            "api-node-port": 9650,
+            "encrypt-connection": false,
+            "account-private-key": "<ACCOUNT_PRIVATE_KEY>"
+        }
+    ]
+}
+```
+
+The following values need to be populated:
+
+<!-- vale off -->
+
+- `<C_CHAIN_BLOCKCHAIN_ID>`: The blockchain ID of the C-Chain. This can be retrieved using [platform.getBlockchains](https://docs.avax.network/reference/avalanchego/p-chain/api#platformgetblockchains) and looking for the blockchain with alias `C`.
+- `<TELEPORTER_CONTRACT_ADDRESS>`: The address of the Teleporter contract on the source and destination chains. This is the same address that Teleporter was deployed to in [Deploy Teleporter](#deploy-teleporter).
+- `<REWARD_ADDRESS>`: The address that the relayer will use to collect rewards for delivering messages. This can be any address.
+- `<ACCOUNT_PRIVATE_KEY>`: The private key of the account that will be used to sign transactions on the source and destination chains. This account must have enough funds to pay for transaction fees on both chains.
+
+<!-- vale on -->
+
+The relayer is now configured to relay messages between the C-Chain and Subnet A. Subnet B can be added as well by extending the `source-subnets` and `destination-subnets` arrays with the appropriate values.
+
+##### Run the Relayer
+
+To run the relayer, we can either build the application from source, or run using a published [Docker image](https://hub.docker.com/r/avaplatform/awm-relayer/tags). For the following examples, assume we've written the relayer configuration to `~/config/config.json` on the host machine.
+
+ To run the relayer built from source:
+
+```bash
+# Clone the repository
+git clone git@github.com:ava-labs/awm-relayer.git
+
+# Build the application
+./scripts/build.sh
+
+# Run the relayer
+./build/awm-relayer --config-file ~/config/config.json
+```
+
+To run the relayer from the Docker image:
+
+```bash
+# Pull the image
+docker pull avaplatform/awm-relayer:latest
+
+# Run the relayer
+docker run --env CONFIG_FILE=/config/config.json -v ~/config:/config avaplatform/awm-relayer:latest
+```
+
+<!-- vale off -->
+
+#### Interact with Teleporter
+
+With the Warp-enabled devnet deployed, Teleporter deployed on each of the chains, and an AWM Relayer instance running, the Teleporter development environment is ready to go! A good place to start are the [example applications](https://github.com/ava-labs/teleporter/blob/main/contracts/src/CrossChainApplications/README.md) included in the Teleporter repository. These dApps are built on top of Teleporter and demonstrate various cross-chain messaging use cases.
```

Comment:
could you add a simplest example, maybe using curl, outside of any other ref, and then link to other docs

Reply:
I think any example included inline here would make this doc unnecessarily messy. Instead, I've linked to the various Teleporter getting started guides and example scripts to serve as a starting point. Specifically, https://github.com/ava-labs/teleporter/blob/main/scripts/local/examples/basic_send_receive.sh includes all the commands for interacting with Teleporter, and serves as documentation in and of itself.